op-faucet/flags/flags.go

```diff
@@ -0,0 +1,71 @@
+package flags
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/ethereum-optimism/optimism/op-faucet/config"
+	fconf "github.com/ethereum-optimism/optimism/op-faucet/faucet/backend/config"
+	opservice "github.com/ethereum-optimism/optimism/op-service"
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	opmetrics "github.com/ethereum-optimism/optimism/op-service/metrics"
+	"github.com/ethereum-optimism/optimism/op-service/oppprof"
+	oprpc "github.com/ethereum-optimism/optimism/op-service/rpc"
+	"github.com/ethereum-optimism/optimism/op-service/txmgr"
+)
+
+const EnvVarPrefix = "OP_FAUCET"
+
+func prefixEnvVars(name string) []string {
+	return opservice.PrefixEnvVar(EnvVarPrefix, name)
+}
+
+var (
+	ConfigFlag = &cli.StringFlag{
+		Name:    "config",
+		Usage:   "Configuration file path",
+		EnvVars: prefixEnvVars("CONFIG"),
+		Value:   config.DefaultConfigYaml,
+	}
+)
+
+var requiredFlags = []cli.Flag{}
+
+var optionalFlags = []cli.Flag{
+	ConfigFlag,
+}
+
+func init() {
+	optionalFlags = append(optionalFlags, oprpc.CLIFlags(EnvVarPrefix)...)
+	optionalFlags = append(optionalFlags, oplog.CLIFlags(EnvVarPrefix)...)
+	optionalFlags = append(optionalFlags, opmetrics.CLIFlags(EnvVarPrefix)...)
+	optionalFlags = append(optionalFlags, oppprof.CLIFlags(EnvVarPrefix)...)
+	optionalFlags = append(optionalFlags, txmgr.CLIFlags(EnvVarPrefix)...)
```

Comment:
may as well move the ConfigFlag into this init() so that preinitialized flags are actually empty

Reply:
Either way is fine I think, other services also define a few in-place like `ConfigFlag` here.